docs/src/app/components/DatePicker/ExampleSimple.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import DatePicker from 'material-ui/lib/date-picker/date-picker';
+import TextField from 'material-ui/lib/text-field';
+import Toggle from 'material-ui/lib/toggle';
+
+const optionsStyle = {
+  width: '300px',
```

Comment:
That's better here :+1:.
Since it's for documentation, I would say that
```js
  width: 300,
```
is better